provisioning/service/src/Microsoft.Azure.Devices.Provisioning.Service.csproj

```diff
@@ -2,13 +2,67 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\..\common\src\service\AssertionFailedException.cs" Link="common\AssertionFailedException.cs" />
```

Comment:
TODO: We need to create a common include csproj which defines a $(Common) that we can use throughout our projects.